### DISCUSSION TITLE
SOnOff : frequency at which the sensor values are sent

### DISCUSSION THREAD

**Discussion by DSC4465** [arendst/Tasmota#11798] [Q&A] [answered]
I would like the SOnOff TH16 to send me the values of the probes (temperature, humidity) every minute. How can I do ? I did not find a parameter to adjust the sending frequency.

**Comment by sfromis** [comment, marked as answer]
The parameter for telemetry frequency is set via the command [TelePeriod](https://tasmota.github.io/docs/Commands/#teleperiod).
